### PR TITLE
DOC-828 Add exists operator to cache processor

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-828](https://redpandadata.atlassian.net/browse/DOC-828)
Review deadline: December 11th

Add `exists` operator.

## Page previews

[`cache` processor](https://deploy-preview-123--redpanda-connect.netlify.app/redpanda-connect/components/processors/cache/)

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-828]: https://redpandadata.atlassian.net/browse/DOC-828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mihaitodor — APPROVED** [review]
LGTM :shipit:

File: modules/components/pages/processors/cache.adoc [hunk 1]
```diff
@@ -225,6 +226,10 @@ Retrieve the contents of a cached key and replace the original message payload
 with the result. If the key does not exist the action fails with an error, which
 can be detected with xref:configuration:error_handling.adoc[processor error handling].
 
+=== `exists`
+
+Check whether a specific key is already cached and replace the original message payload with `true` if the key exists in the cache, or `false` if it doesn't.
```
Comment (Deflaimun) [inline modules/components/pages/processors/cache.adoc:231]:
```suggestion
Check if a specific key is in the cache and set the message payload to `true` if it exists, or `false` if it doesn’t.
```

Suggest this simplification.

**Review by Deflaimun — APPROVED** [review]
See suggestion

File: modules/components/pages/processors/cache.adoc (hunk 1 above)
Comment (asimms41) [reply modules/components/pages/processors/cache.adoc:231]: Updated but kept consistent with other descriptions.

**Pull request merged**